src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  box-sizing: border-box;
+  height: 100vh;
+  background: #e5e5e5;
+}
+
+.logo-link {
+  margin-left: 50px;
+  height: 40px;
+  text-decoration: none;
+}
+
+.header {
+  max-width: 1200px;
+  min-width: 1024px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+}
+
+.nav {
+  margin-right: 50px;
```

Comment:
instead you can add paddings to your header

Reply:
I tried to do this from the beginning. But npm test treated it as an error.